packaging/hoodie-spark-bundle/src/main/java/com/uber/hoodie/spark/bundle/Main.java

```diff
@@ -0,0 +1,52 @@
+/*
+ *  Copyright (c) 2017 Uber Technologies, Inc. (hoodie-dev-group@uber.com)
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *           http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *
+ */
+
+package com.uber.hoodie.spark.bundle;
+
+import com.google.common.reflect.ClassPath;
+import java.lang.String;
+import java.io.IOException;
+
+/**
+ * A simple main class to dump all classes loaded in current classpath
+ *
+ * This is a workaround for generating sources and javadoc jars for packaging modules.
+ * The maven plugins for generating javadoc and sources plugins do not generate
+ * corresponding jars if there are no source files.
+ *
+ * This class does not have anything to do with Hudi but is there to keep mvn javadocs/source
+ * plugin happy.
+ */
+public class Main {
+
+  public static void main(String[] args) {
+    dumpTopLevelClasses(Main.class);
+  }
+
+  private static void dumpTopLevelClasses(Class clazz) {
+    try {
+      ClassPath classPath = ClassPath.from(clazz.getClassLoader());
```

Comment:
anyway to share this code across the 3 classes?  for e.g: it can be a util method in ReflectUtils?

Reply:
Sure, Will do